nextnanopy/inputs.py

```diff
@@ -746,15 +748,15 @@ def execute_sweep(self, delete_input_files = False, overwrite = False, show_log
             warnings.warn('Nothing was executed in sweep! Input files to execute were not created.')
 
         #TODO: delete if statement (use execution_queue for both cases)
-        #TODO: add "Executig simulations" to ExecutionQueue if not show_log
         if parallel_limit>1:
             execution_queue = ExecutionQueue(limit_parallel=parallel_limit, terminate_empty=True, outputdirectory = output_directory, show_log = show_log, convergenceCheck = convergenceCheck, convergence_check_mode = convergence_check_mode, **kwargs)
             execution_queue.add(*self.input_files)
             execution_queue.start()
 
             execution_queue.join()
             if delete_input_files:
-                inputfile.remove()
+                for inputfile in self.input_files:
+                    inputfile.remove()
         else:
             for i, inputfile in enumerate(self.input_files):
                 if not show_log:
```

Comment:
"Executing simulations..." statement can be replaced by "Remaining simulations in the queue" if we use ExecutionQueue for both cases parallel_limit>1 and =1.